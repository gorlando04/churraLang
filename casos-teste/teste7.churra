amigo "Ana" contribui 80 vegetariano
amigo "Bia" contribui 80
item carne "costela" preco_kg 40 quantidade_kg 2
item bebida preco_L 20 quantidade_L 2
rateio igual
