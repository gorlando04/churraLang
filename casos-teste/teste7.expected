Total do churrasco: R$120.00
Ana deve pagar: R$20.00
Bia deve pagar: R$100.00